lib/requests/foods.js

```diff
@@ -67,7 +67,7 @@ const renderFood = (food) => {
       <p class="food-item-name">${food.name}</p>
       <p class="food-item-calories">${food.calories}</p>
       <div class="button-container">
-        <button id="${food.id}" class="food-item-delete-btn" aria-label="Delete">-</button>
+        <button id="food-item-${food.id}" class="food-item-delete-btn" aria-label="Delete">-</button>
```

Comment:
This change has already been made into master and committed here: https://github.com/izaxveta/quantified-self/commit/e8f978d9857cbf4cbc0b7ff021950c487e61a995